extensions/Worker.Extensions.Storage.Tables/src/TableInputAttribute.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables
```

Comment:
Should the namespace be `Microsoft.Azure.Functions.Worker`. Check other extensions.